CONTRIBUTING.md

```diff
@@ -25,12 +25,15 @@ again.
 ## Coding style
 
 ### SDK
-See the [SDK-specific Contribution Guidelines](https://github.com/kubeflow/pipelines/blob/master/sdk/README.md) for contribuying to the `kfp` SDK.
+See the [SDK-specific Contribution Guidelines](https://github.com/kubeflow/pipelines/blob/master/sdk/CONTRIBUTING.md) for contribuying to the `kfp` SDK.
```

Comment:
Good catch!

Also, "contributing" -> "contribuying" :)

Reply:
I will update this.